### PR TITLE
fix(Structured data improvements) : coverage start and end time for live blog and date created and published for news article

### PR DESCRIPTION
**Pull request by Harshith46**

### CODE REVIEW

File: src/structured-data/structured-data-tags.js [hunk 1]
```diff
@@ -84,7 +84,7 @@ function generateArticleData (structuredData = {}, story = {}, publisherConfig =
     "author": authorData(authors),
     "keywords": metaKeywords,
     "articleBody": (storyKeysPresence && getCompleteText(story)) || '',
-    "dateCreated": stripMillisecondsFromTime(new Date(story['created-at'])),
+    "dateCreated": stripMillisecondsFromTime(new Date(story['first-published-at'])),
     "dateModified": stripMillisecondsFromTime(new Date(story['updated-at'])),
```
Comment (sinujohn91) [inline src/structured-data/structured-data-tags.js]: this would be `last-published-at` right?

File: src/structured-data/structured-data-tags.js [hunk 2]
```diff
@@ -141,8 +141,8 @@ function findStoryElementField(card, type, field, defaultValue) {
 
 function generateLiveBlogPostingData (structuredData = {}, story = {}, publisherConfig = {}){
   return {
-    "coverageEndTime": stripMillisecondsFromTime(new Date(story['last-published-at'])),
-    "coverageStartTime": stripMillisecondsFromTime(new Date(story['created-at'])),
+    "coverageEndTime": stripMillisecondsFromTime(new Date(story['updated-at'])),
```
Comment (sinujohn91) [inline src/structured-data/structured-data-tags.js]: dont think this change is required.

File: src/structured-data/structured-data-tags.js [hunk 3]
```diff
@@ -158,16 +158,18 @@ function generateLiveBlogPostingData (structuredData = {}, story = {}, publisher
...
     "author": authorData(story.authors),
     "keywords": metaKeywords,
-    "dateCreated": stripMillisecondsFromTime(new Date(story['created-at'])),
+    "dateCreated": stripMillisecondsFromTime(new Date(story['first-published-at'])),
     "dateModified": stripMillisecondsFromTime(new Date(story['updated-at'])),
     "description": story.summary,
     "name": story.headline,
     "thumbnailUrl": [imageUrl(publisherConfig, story['hero-image-s3-key'])],
-    "uploadDate": stripMillisecondsFromTime(new Date(story['published-at']))
+    "uploadDate": stripMillisecondsFromTime(new Date(story['published-at'])),
```
Comment (sinujohn91) [inline src/structured-data/structured-data-tags.js]: just to be more explicit use `last-published-at`

File: src/structured-data/structured-data-tags.js (hunk 1 above)
Comment (Harshith46) [reply src/structured-data/structured-data-tags.js]: jeevan and anand said be to use updated at

File: src/structured-data/structured-data-tags.js (hunk 2 above)
Comment (Harshith46) [reply src/structured-data/structured-data-tags.js]: created and update time provide same timestamp. it is requested that it shouldn't be same

**Pull request merged**